.travis.yml

```diff
@@ -1,9 +1,9 @@
-
 language: php
 
 php:
-  - '7.0'
-  - '7.2.0RC5'
+  - 7.0
+  - 7.1
```

Comment:
do we really need to test on 7.1 if we test the lowest version 7.0 and the newest version 7.2? Worried about build time for no gain

Reply:
I've removed version 7.1 from the commit.